tools/ld-analyse/chromadecoderconfigdialog.cpp

```diff
@@ -44,6 +44,9 @@ ChromaDecoderConfigDialog::ChromaDecoderConfigDialog(QWidget *parent) :
 
     ui->yNRHorizontalSlider->setMinimum(0);
```

Comment:
How about having a single Luma NR slider at the top, under the chroma gain slider? (The Chroma NR slider would need to stay in the NTSC panel for now since there isn't an equivalent for PAL.)